models/ampleforth/ethereum/ampleforth_ethereum_airdrop_claims.sql

```diff
@@ -0,0 +1,45 @@
+{{
+    config(
+        alias='airdrop_claims',
+        materialized = 'incremental',
```

Comment:
based on [this](https://github.com/duneanalytics/spellbook/pull/2979#discussion_r1149317545) comment, can we make this `'table'` in addition to the `static` tag you've added? this should apply to a few of the spells here